pkg/controllers/pushsecret/pushsecret_controller.go

```diff
@@ -137,6 +139,10 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 	}
 	syncedSecrets, err := r.PushSecretToProviders(ctx, secretStores, ps, secret, mgr)
 	if err != nil {
+		if err.Error() == errGetSecretClient{
+			log.Info("skipping unmanaged store as it points to a unmanaged controllerClass")
+			return ctrl.Result{}, nil
```

Comment:
If there is an error we should return it so we'll retry with backoff. Otherwise the PushSecret stays in its broken state until the controller is restarted or the resource is modified.